✨[FEAT] : 설정, 계정 페이지 UI 작업

yuujjeong opened the pull request:
## 📌 관련 이슈
#5  

## 📌 변경 사항 및 이유
<!-- 변경한 내용과 그 이유를 적어주세요. -->
설정 뷰 UI
서비스 이용 - 계정 뷰 UI


## 📌 PR Point
<!-- 리뷰어 분들이 집중적으로 보셨으면 하는 내용을 적어주세요 -->
![image](https://github.com/U-is-Ni-in-Korea/iOS-United/assets/128811092/fc11748e-5e97-4f23-9da1-4db28a274eb1)
![image](https://github.com/U-is-Ni-in-Korea/iOS-United/assets/128811092/034af635-9d0d-493e-80de-4786cb421f54)

yuujjeong committed:
✨[FEAT] : 설정, 계정 페이지 UI 작업

parkikbum commented on Uni/Source/Scene/Mypage/Cells/AccountTableViewCell.swift:
```diff
@@ -0,0 +1,57 @@
+        super.setSelected(selected, animated: animated)
+
+        // Configure the view for the selected state
+    }
```
p1. 사용하지 않는 코드 삭제해주세요!

parkikbum commented on Uni/Source/Scene/Mypage/Cells/AccountTitle.swift:
```diff
@@ -0,0 +1,20 @@
+//  Created by 홍유정 on 2023/07/14.
+//
+
+struct AccountTitle {
+    let title: String
+}
```
구조체로 빼놓거 칭찬해요!!

parkikbum commented on Uni/Source/Scene/Mypage/Cells/SettingTableViewCell.swift:
```diff
@@ -0,0 +1,66 @@
+
+class SettingTableViewCell: UITableViewCell {
+    
+    static let idf = "SettingTableViewCell"
```
p0. 우리는 NSObject+ 파일에 reusableIdentifier 익스텐션이 존재해요!
그래서, 이런식으로 static 변수를 사용해줄 필요가 없답니다~

parkikbum commented on Uni/Source/Scene/Mypage/Cells/SettingTableViewCell.swift:
```diff
@@ -0,0 +1,66 @@
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+        
+    }
```
이친구도 날려주세요~

parkikbum commented on Uni/Source/Scene/Mypage/View/AccountView.swift:
```diff
@@ -0,0 +1,70 @@
+import SDSKit
+import Then
+
+class AccountView: UIView {
```
p1 우리 다시는 상속이 필요없을거 같은 칭구들은 final 붙히는 습관 들여봐요~!

parkikbum committed:
✅[CHORE]: 코드 리뷰 반영

🔨[FIX]: 버그, 오류 해결
➕[ADD]: Feat 이외의 부수적인 코드 추가, 라이브러리 추가, 새로운 파일 생성 시
✨[FEAT]: 새로운 기능 구현
✅[CHORE] : 코드 수정, 내부 파일 수정
⚰️[DEL] : 쓸모없는 파일,코드 삭제
♻️[REFACTOR] : 전면 수정이 있을 때 사용합니다
 🔀[MERGE]: 다른브렌치를 merge 할 때 사용합니다.

parkikbum committed:
Merge branch 'develop' into feat_5_hong
